cpp/src/icegriddb/IceGridDB.cpp

```diff
@@ -47,13 +47,16 @@ namespace
     };
 }
 
+/*
+TODO: how do we deal with this?
```

Comment:
Benoit, what can we do here?

Reply:
I removed support for importing IceGrid 3.5 files - that's the simplest solution.